contracts/account/README.adoc

```diff
@@ -4,10 +4,6 @@ NOTE: This document is better viewed at https://docs.openzeppelin.com/community-
 
 This directory includes contracts to build accounts for ERC-4337. These include:
 
- * {Account}: An ERC-4337 smart account implementation that includes the core logic to process user operations.
```

Comment:
Very minor & pretty out-of-scope:
https://github.com/OpenZeppelin/openzeppelin-community-contracts/blob/f84f33d38431daef9ab848c668e3a58c08fa1dd9/contracts/access/README.adoc?plain=1#L4
to be updated in `https://docs.openzeppelin.com/community-contracts/access`.